install_sedumi.m

```diff
@@ -36,6 +36,13 @@ function install_sedumi( varargin )
 
 need_rebuild = any( strcmp( varargin, '-rebuild' ) );
 no_path = any( strcmp( varargin, '-nopath' ) );
+openblas_path = '';
+for v = 1:length(varargin)
+  [ob_S, ob_E, ob_TE, ob_M, ob_T, ob_NM, ob_SP] = regexp( varargin{v}, '-obpath=(?<path>.*)' );
```

Comment:
Many return values are unused.  Maybe:

```
ob_NM = regexp( blab{v}, '-obpath=(?<path>.*)', 'names' );
```

Reply:
Yes, I was still wondering if any of the other values could be useful, but I ended up in only using the named tokens. If you can confirm that your change works across Octave versions, I will include it.